src/android/build-extras.gradle

```diff
@@ -20,7 +20,9 @@
        under the License.
  */
 ext.postBuildExtras = {
-    def inAssetsDir = file("assets")
+    
+    // Since cordova-android 7 assets directory moved to android/app/src/main/assets instead of android/assets
+    def inAssetsDir = file("src/main/assets").exists() ? file("src/main/assets") : file("assets")
```

Comment:
I'd like to avoid of using code duplications, this could be changed like this:
```java
def inAssetsDir = file("src/main/assets");
if (!inAssetsDir.exists()) { // Add support for cordova-android < 7.0.0
    inAssetsDir = file("assets");
}
```